sdk/storage/azure-storage-file-share/tests/test_file.py

```diff
@@ -1,38 +1,38 @@
-# coding: utf-8
-
 # -------------------------------------------------------------------------
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License. See License.txt in the project root for
 # license information.
 # --------------------------------------------------------------------------
 import base64
 import os
+import requests
```

Comment:
nit: `requests` is actually an external lib and therefore should down above `pytest`.